src/base/rss/private/rss_parser.cpp

```diff
@@ -33,16 +33,329 @@
 
 #include <QDebug>
 #include <QDateTime>
+#include <QGlobalStatic>
+#include <QHash>
 #include <QMetaObject>
 #include <QRegExp>
 #include <QStringList>
 #include <QVariant>
+#include <QXmlStreamEntityResolver>
 #include <QXmlStreamReader>
 
 #include "../rss_article.h"
 
 namespace
 {
+    using StringHash = QHash<QString, QString>;
+    Q_GLOBAL_STATIC_WITH_ARGS(StringHash, HTML_ENTITIES, ({
+        {"nbsp",   "&#160;"}, /* no-break space = non-breaking space, U+00A0 ISOnum */
```

Comment:
If you get this table from somewhere, can you put a url pointing to it?
Also I would use single-line comment: `//` whenever possible, because `/**/` comment cannot nest.

Reply:
Done.